### ISSUE TITLE
Don't push tags as separate command, push everything together

### ISSUE THREAD

**Issue by gpean** [webpro/release-it#104]
Currently `release-it` will `git push` tags **after** it has pushed the new commit:

```
[execute] git tag  --annotate --message="Release 0.0.29" 0.0.29
[execute] git push
To ssh://stash.corp.netflix.com:7999/dta_node/shader_transpiler_service.git
   e99283b..f34c8da  master -> master
[execute] git push --follow-tags
To ssh://stash.corp.netflix.com:7999/dta_node/shader_transpiler_service.git
 * [new tag]         0.0.29 -> 0.0.29
```

This is an issue if you have stuff which is triggered when you push, like a Jenkins job, because it will not know about the tag since it might still be in flight/not yet pushed.

There should be only one git push command done to push both the commit and the new tag: `git push --tags`.

**Comment by webpro** [comment]
I agree, just released this in [v2.8.3](https://github.com/webpro/release-it/releases/tag/2.8.3).

**Comment by webpro** [comment]
It should be `--follow-tags` (`--tags` pushes _only_ tags).

[Bugfix](https://github.com/webpro/release-it/releases/tag/2.8.5).
